Tiltfile.dev

```diff
@@ -1,5 +1,5 @@
 custom_build(
-  ref = '169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/lfp-pay-api',
+  ref = '169942020521.dkr.ecr.eu-west-2.amazonaws.com/local/lfp-pay-api',
```

Comment:
Future task to update this once we have pipelines and build images renamed.

Reply:
Yep, this corresponds to the current private repo image we have in AWS Development.

This file can probably be removed shortly as I believe the `Tiltfile.dev` was needed for an old way of working locally in Tilt which is deprecated.